oauth-server/src/main/java/de/adorsys/oauth/server/TokenStore.java

```diff
@@ -35,6 +35,8 @@
 
     void addRefreshToken(RefreshToken token, UserInfo userInfo, ClientID clientId, LoginSessionToken sessionId);
 
+    void addRefreshToken(RefreshToken token, UserInfo userInfo, ClientID clientId, LoginSessionToken sessionId, long refreshLifeTime);
```

Comment:
mir gefällt mehr ``refreshTokenLifetime`` statt ``refreshLifeTime``
So wird es klar, was die Bedeutung ist.
Sonnst kann ich vorstellen, dass jmd es wie "Time to request new token" auslegen kann